coordinator/src/admin.rs

```diff
@@ -144,6 +156,98 @@ pub async fn list_dlc_channels(
     Ok(Json(dlc_channels))
 }
 
+pub async fn collaborative_revert(
+    State(state): State<Arc<AppState>>,
+    revert_params: Json<CollaborativeRevert>,
+) -> Result<Json<String>, AppError> {
```

Comment:
🔧 Moving the actual logic into a separate function would simplify the error handling and improve maintainability. IMHO we shouldn't have much logic in the API layer.

Reply:
[`7f32d67` (#1478)](https://github.com/get10101/10101/pull/1478/commits/7f32d6783948fa50602024f897ad4168250d31c9)